ui/src/components/auth/country.json

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
i dont think this entire file is necessary? you should be using the country list in lists.json